service/app/commands/handler_rooms_alias_register.go

```diff
@@ -131,7 +131,7 @@ func (h *RoomsAliasRegisterHandler) registerAlias(ctx context.Context, cmd Rooms
 		return nil, errors.New("channel closed")
 	}
 
-	if response.Err != nil {
+	if err := response.Err; err != nil {
```

Comment:
Let's ensure that `ErrRemoteEnd` is not a typical case we should handle as a no-error.